Added explanation on what QueryType is when it's introduced (#1159)
ezsystems/developer-documentation: docs/tutorials/platform_beginner/5_display_a_list_of_content_items.md

The `rides` variable you use in line 15 above needs to contain a list of all Rides.
To get this list, you will use a Query Type.

## QueryType overview

QueryType objects are used to limit and sort results for Content Item queries. 

> They can also define parameters to dynamically control them, allowing changing the resultset based on external factors.

Author: Changed "they" to "QueryTypes", removed the second part about control.

As an example you can allow sorting direction to be controlled by URL parameter (`order=DESC`) or narrow the results.
QueryType objects can also dictate which Content Type objects are queried.

> You can read more about them [here](../../guide/controllers/#creating-custom-query-types)

Reviewer: ```suggestion
For more information about QueryTypes, see [Built-In Query Types](../../guide/controllers/#built-in-query-types)
```

## Create a QueryType for the home page

> In our case we want to display `ride` objects that have been published (are visible).

Reviewer: ```suggestion
Here, you want to display `ride` objects that have been published (are visible).
```
